src/App.tsx

```diff
@@ -4,10 +4,13 @@ import NameForm from './components/pages/StatedNameForm';
 import DashboardPage from './components/pages/StatedGraphForm';
 import './styled/index.css';
 import RoutingService from './components/pages/Routing';
+import GlobalStyle from './styled/globalStyle';
 
 class App extends React.Component<{}, {}> {
   render() {
     return (
+      <>
+      <GlobalStyle />
```

Comment:
Instead of making it separate element you can use GlobalStyle instead of <></> and wrap the code using it.
It will be more logical at least for me that it applies to all elements.

Reply:
GlobalStyle component can't include children becouse it cant render it. 
Info from docs below.
"Returns a StyledComponent that does not accept children. Place it at the top of your React tree and the global styles will be injected when the component is "rendered"

Reply:
ok